com.unity.multiplayer.mlapi/Runtime/Profiling/ProfilerConstants.cs

```diff
@@ -13,12 +13,12 @@ public static class ProfilerConstants
         public const string ByteReceived = nameof(ByteReceived);
         public const string NetworkVarDeltas = nameof(NetworkVarDeltas);
         public const string NetworkVarUpdates = nameof(NetworkVarUpdates);
-        public const string RpcSent = nameof(RpcSent);
-        public const string RpcReceived = nameof(RpcReceived);
-        public const string RpcBatchesSent = nameof(RpcBatchesSent);
-        public const string RpcBatchesReceived = nameof(RpcBatchesReceived);
-        public const string RpcQueueProcessed = nameof(RpcQueueProcessed);
-        public const string RpcInQueueSize = nameof(RpcInQueueSize);
-        public const string RpcOutQueueSize = nameof(RpcOutQueueSize);
+        public const string MessagesSent = nameof(MessagesSent);
+        public const string MessagesReceived = nameof(MessagesReceived);
```

Comment:
Message != RPC right? Because this will affect the front facing part of the profiler. I understand the RPC is going through the message system, but it's still the RPC we want to track.